### PR TITLE
VM: Implement INC, DEC opcode 

### LINKED ISSUE #220: VM: implement INC, DEC opcode

### PR DESCRIPTION
**Pull request by dauTT**
Closes #220

### CODE REVIEW

File: pkg/vm/vm_ops_maths.go [hunk 1]
```diff
@@ -42,6 +44,56 @@ func Sub(op stack.Instruction, ctx *stack.Context, istack *stack.Invocation, rst
 	return NONE, nil
 }
 
+// Inc increments the stack Item's value of 1.
```
Comment (decentralisedkev) [inline pkg/vm/vm_ops_maths.go]: Nit: small typo inc increments the stack item's value by one

File: pkg/vm/vm_ops_maths.go [hunk 2]
```diff
@@ -42,6 +44,56 @@ func Sub(op stack.Instruction, ctx *stack.Context, istack *stack.Invocation, rst
...
+	if err != nil {
+		return FAULT, err
+	}
+
+	ctx.Estack.Push(res)
+
+	return NONE, nil
+}
+
+// Dec decrements the stack Item's value of 1.
```
Comment (decentralisedkev) [inline pkg/vm/vm_ops_maths.go]: same here, but apart from that looks perfect

File: pkg/vm/vm_ops_maths.go (hunk 1 above)
Comment (dauTT) [reply pkg/vm/vm_ops_maths.go]: :+1:

File: pkg/vm/vm_ops_maths.go (hunk 2 above)
Comment (dauTT) [reply pkg/vm/vm_ops_maths.go]: :+1:

**Comment by decentralisedkev** [comment]
LGTM

**Comment by hal0x2328** [comment]
100 points dauTT

**Comment by lllwvlvwlll** [comment]
https://neoscan.io/transaction/630c74684cc6dad330397c63e8c7aa6403c043cc1f241638cff87e3b97a40716

**Pull request merged**